terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -239,6 +222,43 @@ export async function listRunners(
   }
 }
 
+/**
+ * converts ec2 instance metadata to RunnerInfo
+ * @param instance
+ * @param awsRegion
+ * @returns
+ */
+function toRunnerInfo(instance: AWS.EC2.Instance, awsRegion: string): RunnerInfo {
+  const getTag = (key: string) => instance.Tags?.find((t) => t.Key === key)?.Value;
+  return {
+    applicationDeployDatetime: getTag('ApplicationDeployDatetime'),
+    awsRegion,
+    az: instance.Placement?.AvailabilityZone?.toLowerCase(),
+    ebsVolumeReplacementRequestTimestamp: getTag('EBSVolumeReplacementRequestTm')
+      ? parseInt(getTag('EBSVolumeReplacementRequestTm')!)
+      : undefined,
+
+    environment: getTag('Environment'),
+    ephemeralRunnerFinished: getTag('EphemeralRunnerFinished')
+      ? parseInt(getTag('EphemeralRunnerFinished')!)
+      : undefined,
+    ghRunnerId: getTag('GithubRunnerID'),
+    instanceId: instance.InstanceId!,
+    instanceManagement: getTag('InstanceManagement'),
+    launchTime: instance.LaunchTime,
+    repositoryName: getTag('RepositoryName'),
+    repositoryOwner: getTag('RepositoryOwner'),
+    runnerTypeLabels: getTag('RunnerTypeLabels') ? getTag('RunnerTypeLabels')?.split(',') : undefined,
+    runnerExtraLabels: getTag('RunnerExtraLabels') ? getTag('RunnerExtraLabels')?.split(',') : undefined,
+    runnerGroupName: getTag('RunngerGroupName'),
+    org: getTag('Org'),
+    repo: getTag('Repo'),
```

Comment:
Are these fields not redundant?

`repositoryName` vs `repo`
`repositoryOwner` vs `org`
`runnerTypeLabels` vs `runnerType`